{"name": "test_003_ue_ip", "status": "failed", "statusDetails": {"message": "AssertionError: can not get ue ip!\nassert False", "trace": "ue_ip = False\n\n    def test_003_ue_ip(ue_ip):\n>       assert ue_ip, \"can not get ue ip!\"\nE       AssertionError: can not get ue ip!\nE       assert False\n\ntest_003_smf_rel_pdu.py:51: AssertionError"}, "attachments": [{"name": "log", "source": "23b37875-8591-43d6-ba2f-59a891bb5b32-attachment.txt", "type": "text/plain"}], "start": 1566997051389, "stop": 1566997051389, "uuid": "6f9e908e-df02-43d2-9b37-bd5405844346", "historyId": "468c1322e81c6605faee1d85cc3f2947", "testCaseId": "a68f68cf1ada0cbcfc82f1d56e6a2085", "fullName": "test_003_smf_rel_pdu#test_003_ue_ip", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_003_smf_rel_pdu"}, {"name": "subSuite"}, {"name": "host", "value": "ubuntu18"}, {"name": "thread", "value": "32033-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_003_smf_rel_pdu"}]}